doc/whats_new/v1.6.rst

```diff
@@ -227,7 +227,9 @@ Changelog
   :pr:`29135` by :user:`Xuefeng Xu <xuefeng-xu>`.
 
 :mod:`sklearn.linear_model`
-...........................
+............................
+-|Fix| :class:`linear_model.LogisticRegressionCV` Fixed sample weight handling for calculation
+  of test scores under  _log_reg_scoring_path :pr:`29419` by :user:`Shruti Nath <snath-xoc>`.
```

Comment:
```suggestion
...........................
- |Fix| :class:`linear_model.LogisticRegressionCV` corrects sample weight handling
  for the calculation of test scores.
  :pr:`29419` by :user:`Shruti Nath <snath-xoc>`.
```

Reply:
I edited Omar's suggestion to also insert a blank line after the section header to be consistent with the style of the rest of the file.